docker.go

```diff
@@ -1369,6 +1369,17 @@ func (p *DockerProvider) ReuseOrCreateContainer(ctx context.Context, req Contain
 		lifecycleHooks:    []ContainerLifecycleHooks{combineContainerHooks(defaultHooks, req.LifecycleHooks)},
 	}
 
+	if p.config.RyukDisabled {
+		// ryuk disabled, ensure the container we are about to reuse is running
+		if c.State != "running" {
+			// found container to reuse, but it is not in state running
+			fmt.Println("Attempting to start: " + req.Name + " in state: " + c.State)
```

Comment:
suggestion: removing this debugging.

Reply:
Committed by mistake when checking that the `c.State` check was doing what I expected...
Removed now.

Reply:
@stevenh I've removed the debugging as you suggested. Would you be up for doing a second pass?